src/utils/query.js

```diff
@@ -19,27 +19,43 @@ class Query {
 	constructor() {
 		this.client = liskAPIInstance;
 		this.handlers = {
-			account: account => this.getAccount(account),
-			block: block => this.getBlock(block),
-			delegate: delegate => this.getDelegate(delegate),
-			transaction: transaction => this.getTransaction(transaction),
+			account: (account, options) => this.getAccount(account, options),
+			block: (block, options) => this.getBlock(block, options),
+			delegate: (delegate, options) => this.getDelegate(delegate, options),
+			transaction: (transaction, options) =>
+				this.getTransaction(transaction, options),
 		};
 	}
 
-	getBlock(input) {
-		return this.client.sendRequest('blocks/get', { id: input });
+	sendRequest(endpoint, parameters, { testnet } = {}) {
+		const originalTestnetSetting = this.client.testnet;
+		const overrideTestnetSetting =
+			typeof testnet === 'boolean' && this.client.testnet !== testnet;
+		const reset = result => {
```

Comment:
The testnet settings (original setting and reset) seem to be good candidates for extra functions in the class.

Reply:
Updated.